topics/geospatial.md

```diff
@@ -39,3 +39,12 @@ Find all locations within a 5 kilometer radius of a given location, and return t
 3) 1) "station:3"
    2) "2.2441"
 ```
+
+Searching for all stations in a custom defined area or an irregularly shaped area by using the `BYPOLYGON` search option.
+In this example, we are searching within the Downtown Oakland area using polygon vertices that enclose this area.
```

Comment:
Do we think an image will help here? If so, I can add an image to illustrate this

Reply:
![Screenshot 2025-06-13 at 3 38 47 PM](https://github.com/user-attachments/assets/1022bd26-6468-432a-b863-770753448921)